src/protocols/http/hpr_http_roaming_downlink_handler.erl

```diff
@@ -0,0 +1,96 @@
+%%%-------------------------------------------------------------------
+%%% @author jonathanruttenberg
+%%% @copyright (C) 2022, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 19. Sep 2022 12:44 PM
+%%%-------------------------------------------------------------------
+-module(hpr_http_roaming_downlink_handler).
```

Comment:
Does there need to be a elli worker for this callback module in `hpr_sup`?

Reply:
Yes, thanks.
I added one.